package.json

```diff
@@ -1,29 +1,55 @@
 {
```

Comment:
The only modifications to the package.json I am likely to accept in a PR is additional dependencies required for the code being contributed.